build.py

```diff
@@ -1740,11 +1751,18 @@ def __check_vs(self, opts):
         if not os.path.exists(vcvars_bat):
             raise Exception("'%s' could not be found. Please check you have Visual Studio installed at '%s' and that it supports the target platform '%s'." % (vcvars_bat, opts.vs_install_path, opts.platform))
 
+        # Add to the environment the gtk paths so meson can find everything
+        self.add_env('INCLUDE', self.gtk_dir + r'\include')

```

Comment:
Yes, you're completely right. 

BTW I finally (once, but I'm confident for the future) been able to do the rebase to squash the commits: I think that my mistake was not using the -f in the final push so git told me that I was not syncronzed with all the refs and then I start to push, pull, fetch, drink & jump, random, to try to fix it, ending up with 10 commits :(